.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -140,7 +140,7 @@ steps:
   - bash: |
         echo MODE=$(MODE) && \
         SIG_IMAGE_NAME="$(cat vhdbuilder/packer/settings.json | grep "sig_image_name" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')" && \
-        gen2_captured_sig_version="$(cat vhdbuilder/packer/settings.json | grep "gen2_captured_sig_version" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')" && \
+        captured_sig_version="$(cat vhdbuilder/packer/settings.json | grep "captured_sig_version" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')" && \
```

Comment:
@wanqingfu @alexeldeib ended up renaming "gen2_captured_sig_version" to "captured_sig_version" since we're now using it in all cases on the linux side, if you don't think this makes sense then I can revert - functionality wise everything should remain the same on the windows side.

Reply:
i'm okay with it. yeah the conversion should function properly.